wire-ios-data-model/Source/Model/ZMManagedObject.m

```diff
@@ -391,7 +391,7 @@ + (instancetype)internalFetchObjectWithRemoteIdentifier:(NSUUID *)uuid
                                   [self remoteIdentifierDataKey], uuid.data,
                                   [self domainKey], domain];
     }
-
+    
```

Comment:
Can we revert these white spaces, because it violates SwiftLint rules

Reply:
yes